src/App.jsx

```diff
@@ -3,6 +3,7 @@ import { Dashboard } from "./components/dashboard/dashboard";
 import { SignIn } from "./components/signIn/signIn";
 import { Register } from "./components/register/register.jsx";
 import { List } from "./components/list/list";
+import Navbar from "./components/Navbar/Navbar.jsx";
```

Comment:
Navbar is showing the sign out button and User logo on the sign in page. Can you fix it so that shows up after user is logged in?